context/src/main/java/io/micronaut/scheduling/annotation/Scheduled.java

```diff
@@ -46,6 +46,13 @@
      */
     String cron() default "";
 
+    /**
+     * A String representation of the {@link java.time.ZoneId} to base our cron expression on.
+     * Defaults to {@link java.time.ZoneId#systemDefault}
+     * @return The timezone to base the cron expression on
+     */
+    String timezone() default "";
```

Comment:
This should probably be `zoneId` or `timezoneId` to make it more clear how it will be used